<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="mainwindowStack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="homePage">
     <widget class="QPushButton" name="testButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>220</y>
        <width>86</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>481</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="iDrinkLabel">
         <property name="minimumSize">
          <size>
           <width>269</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>269</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Droid Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Liberation Sans'; font-size:24pt;&quot;&gt;iDrink &lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Sans'; font-size:24pt; vertical-align:super;&quot;&gt;TM&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="creditValueLabel">
         <property name="minimumSize">
          <size>
           <width>269</width>
           <height>77</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>269</width>
           <height>77</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="settingsButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
     <widget class="QLabel" name="warningLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>260</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Warning! System Error, contact employee.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="drinkpickerPage">
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>141</width>
        <height>56</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="creditLabel_2">
         <property name="text">
          <string>Credit:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="creditValueLabel_2">
         <property name="text">
          <string>0</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>214</width>
        <height>229</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="drinkButton1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="text">
            <string>Drink 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="drinkButton2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="text">
            <string>Drink 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="drinkButton3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="text">
            <string>Drink 3</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="drinkButton4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="text">
            <string>Drink 4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="drinkButton5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="text">
            <string>Drink 5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="drinkButton6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="text">
            <string>Drink 6</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="drinkPickerHomeButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>71</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>71</height>
       </size>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QLabel" name="expensiveLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>That drink is too expensive for you..</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pleaseWaitPage">
     <widget class="QLabel" name="pleaseWaitLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Please Wait..&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="servicePage">
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>103</width>
        <height>227</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="replaceBottlesButton">
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>71</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>71</height>
          </size>
         </property>
         <property name="text">
          <string>Replace Bottles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="systemSettingsButton">
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>71</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>71</height>
          </size>
         </property>
         <property name="text">
          <string>System Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>0</y>
        <width>20</width>
        <height>271</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="serviceHomeButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>71</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>71</height>
       </size>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QLabel" name="iDrinkLabel_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;iDrink&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="replaceBottlesPage">
     <widget class="QPushButton" name="settingsBackButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>71</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>71</height>
       </size>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="Line" name="line_7">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>0</y>
        <width>20</width>
        <height>271</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;When the Bottle is replaced,&lt;/p&gt;&lt;p align=&quot;center&quot;&gt; please click the button accordingly.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>214</width>
        <height>229</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QPushButton" name="bottleButton">
           <property name="minimumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="text">
            <string>Bottle1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bottleButton_4">
           <property name="minimumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="text">
            <string>Bottle3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bottleButton_6">
           <property name="minimumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="text">
            <string>Bottle5</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QPushButton" name="bottleButton_2">
           <property name="minimumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="text">
            <string>Bottle2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bottleButton_3">
           <property name="minimumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="text">
            <string>Bottle4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bottleButton_5">
           <property name="minimumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>101</width>
             <height>71</height>
            </size>
           </property>
           <property name="text">
            <string>Bottle6</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="systemSettingsPage">
     <widget class="QPushButton" name="settingsBackButton_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>71</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>71</height>
       </size>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="Line" name="line_8">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>0</y>
        <width>20</width>
        <height>271</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>131</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Database Address:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="addressBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>167</width>
        <height>110</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QPushButton" name="testDBButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Test Database</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="testPsocButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Test PSoC</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="DBStatusLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="PsocStatusLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="systemStatusPage"/>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
